src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -284,8 +282,14 @@
               "PartName": "AutoroutePart",
               "Name": "AutoroutePart",
               "Settings": {
-                "ContentTypePartSettings": {
-                  "Position": "2"
+                "Position": "1",
+                "ContentIndexSettings": {
```

Comment:
This `Position` here is wrong, it needs to be as before `ContentTypePartSettings`